cfme/tests/configure/test_tag.py

```diff
@@ -115,7 +115,7 @@ def vm(self, request, a_provider, appliance):
         return _vm(request, a_provider, appliance.rest_api)
 
     @pytest.mark.tier(2)
-    def test_edit_tags(self, appliance, tags):
+    def test_edit_tags_rest(self, appliance, tags):
```

Comment:
OPTIONAL: the test is in the ``TestTagsViaREST``class meaning the current complete testcase name is ``TestTagsViaREST.test_edit_tags``. But I can live with that redundant ``_rest`` suffix.